tensorflow_encrypted/config.py

```diff
@@ -56,6 +57,103 @@ def get_docker_cpu_quota() -> Optional[int]:
     return cpu_cores
 
 
+class TFESession():
```

Comment:
Can we subclass `tf.Session`?

Reply:
It's more of a wrapper around a `tf.session` currently. 

Do you think we can gain something from inheritance?

Reply:
Not a whole lot. Just thought we wouldn't have to implement the `__enter__` and `__exit__` methods. Not a big deal either way.

Reply:
Maybe that wouldn't even work. NVM!!

Reply:
maybe, I will try